apache-rat-plugin/src/test/java/org/apache/rat/mp/RatCheckMojoTest.java

```diff
@@ -200,7 +200,7 @@ public void testIt5() throws Exception {
         assertNotNull(LicenseFamilySetFactory.search("YAL", config.getLicenseFamilies(LicenseFilter.ALL)));
         ReportConfigurationTest.validateDefaultLicenses(config, "MyLicense", "CpyrT", "RegxT", "SpdxT", "TextT", 
                 "Not", "All", "Any");
-        assertNotNull(LicenseSetFactory.search("MyLicense", config.getLicenses(LicenseFilter.ALL)));
+        assertTrue(LicenseSetFactory.search("YAL", "MyLicense", config.getLicenses(LicenseFilter.ALL)).isPresent());
```

Comment:
Could we use assertJ's assertThat(...).isPresent here?

Reply:
np, I just thought that 
https://javadoc.io/static/org.assertj/assertj-core/3.24.2/org/assertj/core/api/AbstractOptionalAssert.html#isPresent()
assertThat(LicenseFactory.search(....)).isPresent() would be easier to read. Thanks.